src/layers/XRLayer/xr-layer-fragment.glsl

```diff
@@ -43,13 +43,16 @@ vec3 rgb2hsv(vec3 rgb) {
  	if (Cmax > Cmin) {
  		hsv.y = delta / Cmax;
 
- 		if (rgb.r == Cmax)
- 			hsv.x = (rgb.g - rgb.b) / delta;
+ 		if (rgb.r == Cmax){
+      hsv.x = (rgb.g - rgb.b) / delta;
+    }
  		else {
- 			if (rgb.g == Cmax)
- 				hsv.x = 2. + (rgb.b - rgb.r) / delta;
- 			else
- 				hsv.x = 4. + (rgb.r - rgb.g) / delta;
+ 			if (rgb.g == Cmax){
+        hsv.x = 2. + (rgb.b - rgb.r) / delta;
+      }
+ 			else{
+        hsv.x = 4. + (rgb.r - rgb.g) / delta;
```

Comment:
minor comment, but could you please format these if/else blocks? I'm guessing prettier doesn't know what to do with  these `.glsl` files.

Reply:
Let me give it another try.  I tried doing one of them but git didn't pick up any diff.

Reply:
When I checkout another branch and then change back to this one, they look ok.  Could you try checking this branch out and seeing if you notice anything wonky?  If not, let me know and I'll merge.